src/InfoboxThemeData.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+import {Model} from './model';
+
+export class InfoboxThemeData extends Model {
+
+	constructor(properties = {}) {
+		super();
+
+		this.items = null;
```

Comment:
And honestly, I would just set them directly on the model. Seems like `InfoboxThemeData` doesn't need any of the methods below: `setItems` or `set`, it can just use the set method inherited from `Model`